custom/enikshay/tests/test_voucher_counts.py

```diff
@@ -155,12 +181,10 @@ def test_voucher_generation_dates(self):
             'drugs_ordered_readable': "Happy Pills, Sad Pills, Buggy Pillz"
         })
         voucher11 = get_voucher_case_structure(None, prescription1.case_id, {
-            'date_fulfilled': '2013-01-01',
```

Comment:
I think this test data is incorrect - shouldn't vouchers have `date_fulfilled` set only when the `state` is set to `fulfilled`?